assets/styles/global.scss

```diff
@@ -559,3 +559,27 @@ a.has-text-grey {
 .is-size-3-5 {
   font-size: 1.6rem;
 }
+
+// ... dots 
```

Comment:
move to `assets/styles/abstract/_animation`

Reply:
done